tests/Integration/PreventEmptySessionCookieIntegrationTest.php

```
@@ -0,0 +1,285 @@
+     */
+    public function testCleanupHandlerNotRegisteredForExistingCookie(): void
+    {
+        $options = new RedisSessionHandlerOptions(null, null, $this->logger);
+        $handler = new RedisSessionHandler($this->connection, new PhpSerializeSerializer(), $options);
+
+        $_COOKIE[session_name()] = 'existing_session_id_' . uniqid();
+
+        PreventEmptySessionCookie::setup($handler, $this->logger);
+
+        $this->addToAssertionCount(1);
+    }
```

PHPMD (2.15.0)

testCleanupHandlerNotRegisteredForExistingCookie accesses the super-global variable $_COOKIE. (undefined)

(Superglobals)